components/Common/Search/index.module.css

```
@@ -0,0 +1,29 @@
+.searchButton {
+  @apply relative
+    w-52
+    rounded-md
+    bg-neutral-100
+    py-2
+    pl-9
+    pr-4
+    text-left
+    text-sm
+    text-neutral-700
+    transition-colors
+    duration-200
+    ease-in-out
+    hover:bg-neutral-200
+    hover:text-neutral-800
+    dark:bg-neutral-900
+    dark:text-neutral-600
+    dark:hover:bg-neutral-800
+    dark:hover:text-neutral-500;
+}
+
+.magnifyingGlassIcon {
+  @apply absolute
+    left-2
+    top-[8px]
+    h-5
+    w-5;
```

```suggestion
    size-5;
```

[tailwind doc](https://tailwindcss.com/docs/size)